app/src/main/java/com/babylon/wallet/android/data/gateway/extensions/EntityMetadataCollectionExtensions.kt

```diff
@@ -208,16 +242,19 @@ fun EntityMetadataItem.toMetadata(): Metadata? = when (val typed = value.typed)
             Metadata.Primitive(
                 key = key,
                 value = "${it.resourceAddress}:${it.nonFungibleId}",
```

Comment:
shouldn't here we also use `NonFungibleGlobalId` as a type, same as for `MetadataNonFungibleGlobalIdValue` above?

Reply:
Oops missed that.